shell/src/main/java/alluxio/cli/docgen/DocValidator.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.docgen;
+
+import difflib.Delta;
+import difflib.DiffUtils;
+import difflib.Patch;
+
+import java.util.List;
+
+/**
+ * Doc validation abstract class to define common methods and constants.
+ */
+abstract class DocValidator {
```

Comment:
not sure why this is an abstract class when there is nothing to extend since everything in this class is static. should it be a utils class?